translations/trait/trait_event.en.yml

```diff
@@ -3,7 +3,7 @@ trait:
 
 title: "Title"
 description: "Description"
-start_date: "Date (begin: 17:00)"
+start_date: "Date (begin: 18:00)"
```

Comment:
18:15?

Reply:
yes, but I thought there was a reason why it was 17:00 in the first place. ima change this.